pydantic/_internal/_core_metadata.py

```diff
@@ -60,16 +60,6 @@ def metadata(self) -> CoreMetadata:
             raise TypeError(f'CoreSchema metadata should be a dict; got {metadata!r}.')
         return metadata  # type: ignore[return-value]
 
-    def get_json_schema(
```

Comment:
same here. there is just one test that was added to cover this function. I've removed both the function and test